src/main/java/edu/cornell/kfs/sys/batch/service/impl/DocumentRouteHeaderValueLookupCriteriaPurgeServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package edu.cornell.kfs.sys.batch.service.impl;
+
+import java.sql.Date;
+
+import org.apache.ojb.broker.query.Criteria;
+import org.kuali.kfs.kew.api.KewApiConstants;
+
+import edu.cornell.kfs.sys.batch.service.TableLookupCriteriaPurgeService;
+
+public class DocumentRouteHeaderValueLookupCriteriaPurgeServiceImpl extends TableLookupCriteriaPurgeServiceImpl
+        implements TableLookupCriteriaPurgeService {
+
+    private static final String DOCUMENT_ROUTE_STATUS = "docRouteStatus";
+    private static final String CREATE_DATE = "createDate";
```

Comment:
the String "createDate" is used in a number of places in our code, but we don't have a constant. Should this be moved into one of our constant classes, and a follow up user story to update the classes to use a constant?

Reply:
Base code KRADPropertyConstants defines the createDate String as a constant Replacing with that base code value instead of creating a local customization for it.
DOCUMENT_ROUTE_STATUS is not defined anywhere in the code base so leaving local to this class.